pkg/agent/route/route_linux.go

```diff
@@ -324,86 +244,77 @@ func (c *Client) Reconcile(podCIDRs []string) error {
 	}
 	for _, entry := range entries {
 		if desiredPodCIDRs.Has(entry) {
+			klog.Infof("Reconcile has entry %s", entry)
 			continue
 		}
-		klog.V(4).Infof("Deleting orphaned ip %s from ipset", entry)
+		klog.Infof("Deleting orphaned ip %s from ipset", entry)
 		if err := ipset.DelEntry(antreaPodIPSet, entry); err != nil {
 			return err
 		}
+		_, cidr, err := net.ParseCIDR(entry)
```

Comment:
Line 254 - 261 are a new fix?

Reply:
Better to add some code comments to explain the logic too.

Reply:
modified log message to reflect the intention.